In-house patch to further adjust Glance's oslo-config-generator files
to account for which modules are actually imported for each service.
These changes are meant as a follow-on to Launchpad bug 1500361 but
they have not yet been submitted upstream.

--- glance-12.0.0/etc/oslo-config-generator/glance-api.conf.~1~	2016-04-07 00:37:11.000000000 -0700
+++ glance-12.0.0/etc/oslo-config-generator/glance-api.conf	2016-07-30 00:39:05.833802230 -0700
@@ -9,4 +9,4 @@ namespace = oslo.db.concurrency
 namespace = oslo.policy
 namespace = keystonemiddleware.auth_token
 namespace = oslo.log
-namespace = oslo.middleware.cors
+namespace = oslo.middleware
--- glance-12.0.0/etc/oslo-config-generator/glance-cache.conf.~1~	2016-04-07 00:37:11.000000000 -0700
+++ glance-12.0.0/etc/oslo-config-generator/glance-cache.conf	2016-07-30 00:39:33.080265565 -0700
@@ -3,3 +3,5 @@ output_file = etc/glance-cache.conf.samp
 namespace = glance.cache
 namespace = oslo.log
 namespace = oslo.policy
+namespace = glance.store
+namespace = oslo.concurrency
--- glance-12.0.0/etc/oslo-config-generator/glance-glare.conf.~1~	2016-04-07 00:37:11.000000000 -0700
+++ glance-12.0.0/etc/oslo-config-generator/glance-glare.conf	2016-07-30 00:39:05.834807800 -0700
@@ -6,4 +6,7 @@ namespace = oslo.db
 namespace = oslo.db.concurrency
 namespace = keystonemiddleware.auth_token
 namespace = oslo.log
-namespace = oslo.middleware.cors
+namespace = oslo.middleware
+namespace = oslo.concurrency
+namespace = oslo.messaging
+namespace = oslo.policy
--- glance-12.0.0/etc/oslo-config-generator/glance-registry.conf.~1~	2016-04-07 00:37:11.000000000 -0700
+++ glance-12.0.0/etc/oslo-config-generator/glance-registry.conf	2016-07-30 00:39:05.835268420 -0700
@@ -8,3 +8,5 @@ namespace = oslo.db.concurrency
 namespace = oslo.policy
 namespace = keystonemiddleware.auth_token
 namespace = oslo.log
+namespace = oslo.concurrency
+namespace = oslo.middleware
